templates/quickstart/foundry/testing/README.md

```diff
@@ -189,6 +189,24 @@ $ tree . -d -L 1
 └── test
 ```
 
+### Private key setup
+
+It's recommended to use Foundry keystore to store your development account private key.
+
+1. Export your account private key.
+2. Create a keystore and import your private key by running:
+
+```shell
+cast wallet import --interactive
+# enter your PK when prompted and provide a password
+```
+
+> Note that `myKeystore` can be any name.
```

Comment:
We could replace this with the sentence we use in the other PR: 

Note that the name `myKeystore` is arbitrary and can be updated. We've chosen this name for consistency.
If you decide to use another name, be sure to reference it when using `cast`.